services/sourceforge/sourceforge-commit-count.service.js

```diff
@@ -14,17 +14,17 @@ export default class SourceforgeCommitCount extends BaseJsonService {
     pattern: ':project',
   }
 
-  static examples = [
-    {
-      title: 'SourceForge commit count',
-      namedParams: {
-        project: 'guitarix',
+  static openApi = {
+    '/sourceforge/commit-count/{project}': {
+      get: {
+        summary: 'SourceForge commit count',
```

Comment:
Nit: we seem to prefer capitalising all words within the badge title, even though we're not consistent across the board. We should probably agree on one or the other. In this particular instance, there's inconsistency within different SourceForge badges themselves, so maybe we could at least fix these? Feel free to dismiss if you don't want to do this now, after all this inconsistency was already present before.